docs/quickstart.md

```diff
@@ -451,3 +451,26 @@ as above:
 >>> httpx.get("https://example.com", auth=auth)
 <Response [200 OK]>
 ```
+
+## Errors and Exceptions
+
+In some situations, HTTPX can raise exceptions:
+
+* `NetworkError` in case of network problem (e.g. DNS failure, refused connection, ...)
+* `ConnectTimeout`, `ReadTimeout`, `WriteTimeout`, `PoolTimeout` in case of a configured timeout is reached. All of them inherit from `TimeoutException`
+* `TooManyRedirects` if a request exceeds the configured number of maximum redirections
+
+Also when `Response.raise_for_status()` is called HTTPX will also `HTTPStatusError` if the HTTP request returned status code >= 400.
+
+All exceptions that HTTPX explicitly raises inherit from httpx.RequestError and contains the request and the response if any.
```

Comment:
A suggestion to clarify that the request is _always_ included (but not the response):

```suggestion
All exceptions that HTTPX raises inherit from `httpx.RequestError` and contain the request, as well as any receved response.
```